indico/modules/events/abstracts/forms.py

```diff
@@ -474,7 +474,13 @@ def __init__(self, *args, **kwargs):
         if abstracts_settings.get(self.event, 'contrib_type_required'):
             inject_validators(self, 'submitted_contrib_type', [DataRequired()])
         super(AbstractForm, self).__init__(*args, **kwargs)
-        self.submitted_contrib_type.query = self.event.contribution_types
+        if kwargs['management']:
```

Comment:
management = kwargs.pop('management')

before calling super (i'd put it after setting `self.abstract`); then use `management` here. no need to pass along  this kwarg to the superclass.

also, it sounds like something where a default value could make sense =>

    management = kwargs.pop('management', False)